core/src/v2/actix.rs

```diff
@@ -277,6 +277,9 @@ impl OperationModifier for actix_multipart::Multipart {
     }
 }
 
+#[cfg(feature = "actix-session")]
+impl OperationModifier for actix_session::Session {}
```

Comment:
We have to override `update_definitions` to an empty function

Reply:
So both `update_security_definitions` and `update_definitions` I think.

Reply:
No, just `update_definitions` would be good, because we haven't implemented any security stuff for that type.

Reply:
done